content/open-source/slsa/what-is-slsa.md

```diff
@@ -16,69 +16,61 @@ weight: 50
-For this level, every change in the revision’s history must be agreed to by two trusted persons prior to submission. The software must also be produced through a hermetic, reproducible build process. 
+Generating provenance compliant with Level 3 requirements can help end users verify the integrity of the software before implementing it. Recently, SLSA released the open source [SLSA 3 Container Generator for GitHub Actions](https://github.com/slsa-framework/slsa-github-generator) that helps ease the process by allowing you to build automated provenance generation into your container workflows. To learn more about how it works, visit the [General availability of SLSA 3 Container Generator for GitHub Actions](https://slsa.dev/blog/2023/02/slsa-github-workflows-container-ga) announcement blog post. You can also check out SLSA's guide on [reaching level 3](https://slsa.dev/get-started#slsa-3).
 
-A _hermetic_ build is defined by a build where all build steps, sources, and dependencies are declared with _immutable references_, or identifiers that are guaranteed to point to an immutable artifact whose integrity is authenticated by a cryptographic hash of its contents. These build steps, sources, and dependencies must also include their transitive counterparts, or dependencies of dependencies. The build steps must also run without network access. 
-
-A _reproducible_ build is one in which the build steps will always reproduce “bit-for-bit” identical output. If software cannot meet this requirement, a justification must be provided. Though the build service may not verify reproducibility, consumers should be able to check its reproducibility and can reject the build if they cannot reproduce the results.
 
 ## SLSA Tools and Practices
 
-SLSA is a relatively new framework and its supporting tools and practices are still actively evolving. Some of the requirements listed in the levels above can be met using popular build and version control systems. More specific requirements may require additional tooling, and SLSA hosts some supporting tools in its [GitHub repository](https://github.com/slsa-framework). As mentioned in the description of Level 3, [SLSA released a tool for automating provenance generation](https://slsa.dev/blog/2023/02/slsa-github-workflows-container-ga) with GitHub Actions in February 2023.  
+SLSA is a relatively new framework and its supporting tools and practices are still actively evolving. Some of the previously listed level requirements can be met using popular build and version control systems. More specific requirements may require additional tooling, and SLSA hosts some supporting tools in its [GitHub repositories](https://github.com/slsa-framework). As mentioned in the description of Level 3, [SLSA released a tool for automating provenance generation](https://slsa.dev/blog/2023/02/slsa-github-workflows-container-ga) with GitHub Actions in February 2023.  
```

Comment:
```suggestion
The SLSA framework and its supporting tools and practices are still actively evolving. Some of the previously listed level requirements can be met using popular build and version control systems. More specific requirements may require additional tooling, and SLSA hosts some supporting tools in its [GitHub repositories](https://github.com/slsa-framework). As mentioned in the description of Level 3, [SLSA released a tool for automating provenance generation](https://slsa.dev/blog/2023/02/slsa-github-workflows-container-ga) with GitHub Actions in February 2023.  
```

Sorry - my comment in your Google Doc wasn't clear. I think we can stop calling it relatively new at this point.